dkobo/koboform/api.py

```diff
@@ -15,10 +25,14 @@ class SurveyAssetViewset(viewsets.ModelViewSet):
     exclude_asset_type = False
 
     def get_queryset(self):
-        user = self.request.user
-        if user.is_anonymous():
-            raise PermissionDenied
-        queryset = SurveyDraft.objects.filter(user=user)
+        myw_kobo_user_cookie = self.request.COOKIES.get('myw_kobo_user')
```

Comment:
can this be more named more generically than `myw_...` like if we want to do the same thing in Ona would we use this piece of functionality?

If so, can we remove the `myw_` prefix for the var and cookie?

If not, can remove the `myw_` prefix for the var and but the cookie text string name in a configuration file?